doc/tutorial.md

```diff
@@ -337,6 +335,16 @@ seastar::future<> f() {
 }
 ```
 
+## Preemption and Task Quota
+
+As described above, an existing fiber of execution will yield back to the event loop when it performs a blocking operation such as IO or sleeping, as it has no more work to do until this blocking operation completes. Should a fiber have a lot of CPU bound work to do without any intervening blocking operations, however, it is important that execution is still yielded back to the event loop periodically.
+
+This is implemented via _preemption_: which can only occur at specific preemption points. At these points the fiber's remaining _task quota_ is checked and it has been exceeded the fiber yields. The task quota is a measure of how long tasks should be allowed to run before yielding to the event loop, and is set to 500 µs by default.
```

Comment:
Maybe we should mention that this is a configurable parameter and how to configure it.

Reply:
I am happy to add this type of thing, but I should interpret your merge as indicating that it is not necessary?